src/pages/ModelDetails/ModelDetails.tsx

```diff
@@ -55,22 +55,11 @@ export default function ModelDetails() {
   }, [modelUUID]);
 
   return (
-    <Switch>
-      <Route path="/models/:userName/:modelName" exact>
-        <Model />
-      </Route>
-      <Route path="/models/:userName/:modelName/app/:appName" exact>
-        <App />
-      </Route>
-      <Route
-        path="/models/:userName/:modelName/app/:appName/unit/:unitId"
-        exact
-      >
-        <Unit />
-      </Route>
-      <Route path="/models/:userName/:modelName/machine/:machineId" exact>
-        <Machine />
-      </Route>
-    </Switch>
+    <Routes>
```

Comment:
Nested routes now need to be relative (this component is rendered inside the main Routes file).